<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget{
background: qlineargradient(
        spread:pad, 
        x1:0, y1:0, 
        x2:1, y2:1, 
        stop:0 rgb(219, 252, 255)
        stop:1 rgb(178, 213, 255)
    );
}</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>150</number>
          </property>
          <property name="topMargin">
           <number>100</number>
          </property>
          <property name="rightMargin">
           <number>150</number>
          </property>
          <property name="bottomMargin">
           <number>100</number>
          </property>
          <item row="0" column="0">
           <widget class="QWidget" name="widget_2" native="true">
            <property name="styleSheet">
             <string notr="true">QWidget{
background: #fff;
border-radius: 30px;
}</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="1" column="0">
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="5" column="0">
                  <widget class="QLabel" name="label">
                   <property name="styleSheet">
                    <string notr="true">QLabel{
color: rgb(88, 88, 88);
font-size: 15px;
}</string>
                   </property>
                   <property name="text">
                    <string>Welcome To RazX OS V1.11</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="styleSheet">
                    <string notr="true">QLabel{
font-size: 80px;
color: rgb(64, 102, 208);
font-weight: bold;
}</string>
                   </property>
                   <property name="text">
                    <string>Hello!</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                   <property name="wordWrap">
                    <bool>false</bool>
                   </property>
                   <property name="textInteractionFlags">
                    <set>Qt::LinksAccessibleByMouse</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="3" column="0">
                <layout class="QGridLayout" name="gridLayout_5">
                 <item row="2" column="0">
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="3" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <item>
                    <spacer name="horizontalSpacer">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="start">
                     <property name="minimumSize">
                      <size>
                       <width>300</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>300</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QPushButton{
background: rgb(65, 104, 211);
color: #fff;
border: 0px;
border-radius: 5px;
}</string>
                     </property>
                     <property name="text">
                      <string>Get Started</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>10</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>10</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLabel{
color: rgb(88, 88, 88);
font-size: 13px
}</string>
                   </property>
                   <property name="text">
                    <string>RazX OS Is Lightest and Beautiful Linux Based Operating System</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="2" column="0">
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>0</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>43</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_7">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="widget_3" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget{
background: qlineargradient(
        spread:pad, 
        x1:0, y1:0, 
        x2:1, y2:1, 
        stop:0 rgb(219, 252, 255)
        stop:1 rgb(178, 213, 255)
    );
}</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <property name="leftMargin">
           <number>150</number>
          </property>
          <property name="topMargin">
           <number>100</number>
          </property>
          <property name="rightMargin">
           <number>150</number>
          </property>
          <property name="bottomMargin">
           <number>100</number>
          </property>
          <item row="0" column="0">
           <widget class="QWidget" name="widget_4" native="true">
            <property name="styleSheet">
             <string notr="true">QWidget{
background: #fff;
border-radius: 30px;
}</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <spacer name="verticalSpacer_6">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>25</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <item>
                    <widget class="QLabel" name="label_6">
                     <property name="styleSheet">
                      <string notr="true">QLabel{
color: rgb(65, 104, 211);
font-weight: bold;
font-size: 40px
}</string>
                     </property>
                     <property name="text">
                      <string>Desktop Theme</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <widget class="QLabel" name="label_7">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>10</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>10</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QLabel{
color: rgb(88, 88, 88);
font-size: 13px;
letter-spacing: 1px
}</string>
                     </property>
                     <property name="text">
                      <string>Install Desktop Theme To Use RazX OS</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_8">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>10</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>10</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QLabel{
color: rgb(88, 88, 88);
font-size: 13px;
letter-spacing: 1px
}</string>
                     </property>
                     <property name="text">
                      <string>This Process Take Few Seconds</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>80</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QProgressBar" name="progressBar">
                   <property name="minimumSize">
                    <size>
                     <width>300</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>300</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: #fff;
background: rgb(171, 171, 171);
border-radius: 5px;
</string>
                   </property>
                   <property name="value">
                    <number>0</number>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QPushButton" name="themeDownload">
                   <property name="minimumSize">
                    <size>
                     <width>300</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>300</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: #fff;
background: rgb(65, 104, 211);
font-size: 13px;
border-radius: 5px</string>
                   </property>
                   <property name="text">
                    <string>Download</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_7">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>26</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
